### DISCUSSION TITLE
Error Installing Agent

### DISCUSSION THREAD

**Discussion by apixler77** [utmstack/UTMStack#736] [Q&A]
When trying to install a Windows agent on version 10.5.2 returns the error "one or more of the required ports are closed. Please open ports 9000 and 50051.

![UTMStack Agent Istall Error](https://github.com/utmstack/UTMStack/assets/166551483/fdeff0fd-6b05-42ee-9833-bb1526e669ff)

OS: Windows 11, Ubuntu 22.04, VMWare Workstation

Version 10.5.2

Update

I have installed 10.5.3. The agent installs with no issues now, but a couple of integrations are not working. Cisco Meraki and Sentinel One. O365 works and is showing up in the Data Sources, but not Meraki and Sentinel One.

**Comment by Kbayero** [comment]
@apixler77 Please make sure you have run the integrations activation command on the machine where your agent is installed, check the logs in the path C:\Program Files\UTMStack\UTMStack Agent\logs\utmstack_agent.log. Also, verify that your Cisco Meraki or Sentinel One sends logs to the correct address (IP of the machine where you installed the agent). Remember, in Meraki's case, you will only receive logs through port 1470 TCP or 514 UDP. In  Sentinel One's case, 7012 TCP or 7012 UDP. Ensure you are sending logs to those ports, and that your firewall is not blocking such sending.

**Reply by apixler77** [reply]
It shows the logs are being sent to the right IP and confirms in the agent.log file, but I have no source showing up in the UTMStack dashboard. Only the one PC with the agent on it and my O365 integration. Is there anywhere else I can check to make sure they are connecting correctly?

**Reply by apixler77** [reply]
Figured out the install Power Shell command was for a previous install with a different IP. I am good.
